.travis.yml

```diff
@@ -8,68 +8,47 @@ php:
 
 env:
   - DRUPAL_CORE=8.1.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
   - DRUPAL_CORE=8.2.x
+    CODE_REVIEW=0
+    SIMPLETEST_DB=mysql://root:@127.0.0.1/og
+    SIMPLETEST_BASE_URL=http://localhost:8888
+  - REVIEW_STANDARD="Drupal"
+    CODE_REVIEW=1
+  - REVIEW_STANDARD="DrupalPractice"
+    CODE_REVIEW=1
```

Comment:
I don't like this, these environment variables are used as identifiers for the individual test runs, this makes the test results hard to read on Travis.

It's better to export the generic environment variables (SIMPLETEST_...) before running the tests, so they don't pollute the UI.

This should be:

```
env:
  - DRUPAL_CORE=8.1.x
  - DRUPAL_CORE=8.2.x
  - CODE_REVIEW=1
```